training/training.module

```diff
@@ -0,0 +1,378 @@
+<?php
+
+/**
+ * Implements hook_menu
+ * @return array
+ */
+function training_menu(){
```

Comment:
) {
add a space between the closing parenthesis and the opening curly bracket
Fix this in all 29 function having this error.
Use Ctrl + F to find them.
https://www.drupal.org/coding-standards#functdecl